cmd/cluster/context.go

```diff
@@ -378,6 +386,22 @@ func (o *contextOptions) generateContextData() (*contextData, []error) {
 		}
 	}
 
+	GetBannedUser := func() {
+		defer wg.Done()
```

Comment:
This is never used, was going to suggest removing it but I can open a follow up to clean all the waitgroup usage up.